.circleci/config.yml

```diff
@@ -279,13 +307,23 @@ jobs:
         default: ""
         type: string
     steps:
-      - checkout # checkout the code in the project directory
-      - run: pyenv global 3.5.2
-      - run: sudo apt-get update -y && sudo apt-get install -y libgflags-dev
+      - pre-steps
+      - install-gflags
+      - install-gtest-parallel
+      - run:
+          name: "Build unit tests"
+          command: |
+            echo "env: $(env)"
+            echo "** done env"
+            ROCKSDBTESTS_START=<<parameters.start_test>> ROCKSDBTESTS_END=<<parameters.end_test>> ROCKSDBTESTS_SUBSET_TESTS_TO_FILE=/tmp/test_list make V=1 -j32 --output-sync=target build_subset_tests
       - run:
-          name: "Build and unit tests on non-shm"
+          name: "Run unit tests in parallel"
           command: |
-            TMPD=/tmp/rocksdb_test_tmp ROCKSDBTESTS_START=<<parameters.start_test>> ROCKSDBTESTS_END=<<parameters.end_test>> make V=1 J=32 -j32 check_some | .circleci/cat_ignore_eagain
+            sed -i 's/[[:space:]]*$//; s/ / \.\//g; s/.*/.\/&/' /tmp/test_list
+            cat /tmp/test_list
+            export TEST_TMPDIR=/tmp/rocksdb_test_tmp
+            /usr/bin/python ../gtest-parallel/gtest-parallel $(</tmp/test_list) --output_dir=/tmp
```

Comment:
I don't see any test-specific console output when this runs. That makes me doubt being able to figure out what failed, almost all the time.

Reply:
Added a patch to print running tasks
![image](https://user-images.githubusercontent.com/33968/94053590-5ff0f900-fd8f-11ea-8da0-2d389ccd54b7.png)